lua/tests/gmod/classes/convar/__tostring.lua

```diff
@@ -0,0 +1,36 @@
+function GetTestConVar( name, value, description ) -- Abusing the loading order :>
+	return CreateConVar( "ConVarTest" .. ( name or "" ), value or "1.2", 0, description or "This is a Test", -10, 10 )
+end
+
+return {
+    groupName = "ConVar:__tostring",
+
+    cases = {
+        {
+            name = "Exists on the ConVar meta table",
+            func = function()
+                local meta = FindMetaTable( "ConVar" )
+                expect( meta.__tostring ).to.beA( "function" )
+            end
+        },
+
+        {
+            name = "Errors when called with a nil self",
+            func = function()
+                local __tostring = FindMetaTable( "ConVar" ).__tostring
+
+                expect( __tostring( nil ) ).to.errWith( [[bad argument #1 to '__tostring' (ConVar expected, got nil)]] )
```

Comment:
`errWith` requires a function so it can `xpcall` or whatever

```suggestion
                local function test()
                    __tostring( nil )
                end

                expect( test ).to.errWith( [[bad argument #1 to '__tostring' (ConVar expected, got nil)]] )
```

Technically you can do:
```lua
                expect( __tostring, nil ).to.errWith( [[bad argument #1 to '__tostring' (ConVar expected, got nil)]] )
```

but because the expectation [uses `unpack`](https://github.com/CFC-Servers/GLuaTest/blob/main/lua/gluatest/expectations/positive.lua#L109) I think your `nil` would be discarded, and then the error would be "`ConVar exepected, got no value`"